app/App.js

```diff
@@ -283,15 +281,15 @@ class App extends React.Component {
     );
   }
 
-  renderVolumeControl (settings) {
+  renderVolumeControl () {
     return (
       <VolumeControls
         fill={this.props.player.volume}
         updateVolume={this.props.actions.updateVolume}
         muted={this.props.player.muted}
         toggleMute={this.props.actions.toggleMute}
         toggleOption={this.props.actions.toggleOption}
-        settings={settings}
+        settings={this.props}
```

Comment:
Is this correct?

Reply:
Nope, breaks the equalizer. Should be `this.props.settings`.